project/AppDependencies.scala

```diff
@@ -6,30 +6,33 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-26"  % "5.3.0",
-    "uk.gov.hmrc" %% "domain"                      % "5.11.0-play-26",
-    "uk.gov.hmrc" %% "http-caching-client"         % "9.5.0-play-26",
+    "com.typesafe.akka" %% "akka-protobuf" % "2.6.14",
```

Comment:
i don't think we are really using this dep ? can you double check and see if you can remove this ?

Reply:
done